duckduckgo/Android
# Remove acra crash reporting from project

Task/Issue URL: https://app.asana.com/0/488551667048375/675946344778516

**Description**:
Removes ACRA crash reporting. We already have crash reporting via the Play Store.

**Steps to test this PR**:
Run the app and ensure that it still runs properly in both debug and release build modes

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
